### PR TITLE
added basic functionality and placeholder events

### PR DESCRIPTION
**Pull request by sriya112**
- the css gets messed up when there's more than 3 events, so have to change that

### CODE REVIEW

File: components/Page_Layout/FutureEventCard.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                src= {props.imgUrl}
+            />
+            
+            <br/><br/>
+            <div className = "overlay">
+                
+                    <br/><br/>
+                    <h5> {props.when} </h5>
+                    <h5> {props.info} </h5>
+
```
Comment (gmundewadi) [inline components/Page_Layout/FutureEventCard.js:18]: remove extra spaces.

File: components/Page_Layout/FutureEventCard.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import '../../css/FutureEventCardStyle.css';
```
Comment (gmundewadi) [inline components/Page_Layout/FutureEventCard.js:1]: Download the extension Prettier, it’ll auto format the code for you whenever you save your file. Use prettier to all of the files in the PR. This ensures that the formatting of our code is consistent

File: components/Page_Layout/FutureEventCard.js (hunk 1 above)
Comment (gmundewadi) [inline components/Page_Layout/FutureEventCard.js:18]: remove extra spaces

File: css/FutureEventCardStyle.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
+.all {
```
Comment (gmundewadi) [inline css/FutureEventCardStyle.css:1]: lets discuss during our meeting if we want the rectangular cards to have a rounded or sharp edge. I’m personally learning towards sharp edges because they look cleaner, but I’m curious to see what the rest of the team thinks

File: components/Page_Layout/Events.js [hunk 4]
```diff
@@ -1,45 +1,77 @@
...
+    //     <h1 className="w3-center">Upcoming Events</h1>
+    //     <br />
+    //     <h4>Bread Basket</h4>
+    //     <p className="w3-text-grey">
+    //       Assortment of fresh baked fruit breads and muffins 5.50
+    //     </p>
+    //     <br />
+
+    //     <h4>Honey Almond Granola with Fruits</h4>
+    //     <p className="w3-text-grey">
```
Comment (gmundewadi) [inline components/Page_Layout/Events.js:17]: delete the commented out code, no need to have it in master

File: components/Page_Layout/Events.js [hunk 5]
```diff
@@ -1,45 +1,77 @@
...
+    //       className="w3-round w3-image w3-opacity-min"
+    //       alt="Menu"
+    //       style={{ width: "100%" }}
+    //     />
+    //   </div>
+    // </div>
+
+    <div className = "all">
+      <FutureEventCard 
+        name = "Event 1"
```
Comment (gmundewadi) [inline components/Page_Layout/Events.js:50]: I thought that we were going to read in this information from a text file?

File: components/Page_Layout/Events.js [hunk 6]
```diff
@@ -1,45 +1,77 @@
...
+        imgUrl = "https://www.clker.com/cliparts/a/Q/1/9/E/J/red-rounded-with-number-1-md.png"
+        when = "01/01/2020"
+        info = "first event info"
+      />
+
+      <FutureEventCard 
+        name = "Event 2"
```
Comment (gmundewadi) [inline components/Page_Layout/Events.js:57]: change the prop `when` to `date`, makes it clearer

File: components/Page_Layout/FutureEvents.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+// import FutureEventCard from "./FutureEventCard";
+
+// export default function FutureEvents() {
+//     return (
```
Comment (gmundewadi) [inline components/Page_Layout/FutureEvents.js:4]: why is this file necessary?

File: components/Page_Layout/Events.js [hunk 8]
```diff
@@ -1,45 +1,77 @@
...
+    //       src="banner.png"
+    //       className="w3-round w3-image w3-opacity-min"
+    //       alt="Menu"
+    //       style={{ width: "100%" }}
+    //     />
+    //   </div>
+    // </div>
+
+    <div className = "all">
+      <FutureEventCard
```
Comment (gmundewadi) [inline components/Page_Layout/Events.js:49]: add a title that says “Upcoming Events”

**Review by gmundewadi — CHANGES_REQUESTED** [review]

**Comment by gmundewadi** [comment]
> the css gets messed up when there's more than 3 events, so have to change that

I actually think that 3 events is fine. Unless if you are envisioning the future events board to have more than just the next 3 upcoming events. @sriya112

**Comment by gmundewadi** [comment]
also, fill out the checkboxes on your card on the kanban board

**Review by pranav01acharya — APPROVED** [review]
